wgsl/Makefile

```diff
@@ -14,6 +14,7 @@ index.pre.html: $(WGSL_SOURCES)
 
 GHC=https://github.com/gpuweb/gpuweb/blob
 index.html: index.pre.html
+	python3 ../tools/check-nfkc.py
```

Comment:
This action is not logically required to build index.html


It recommend making a 'check-nfkc' a .PHONY target and then have index.html *depend* on check-nkfc.
That way I can run the check independently.

Reply:
Done, thanks!